core/indexer/elasticProcessor.go

```diff
@@ -60,8 +60,15 @@ func NewElasticProcessor(arguments ArgElasticProcessor) (ElasticProcessor, error
 		arguments.AddressPubkeyConverter,
 		arguments.ValidatorPubkeyConverter,
 		arguments.FeeConfig,
+		arguments.ShardCoordinator,
+		arguments.IsInImportDBMode,
 	)
 
+	if arguments.IsInImportDBMode {
+		log.Warn("the node is in import mode! Elastic search will not send cross shard transactions and rewards " +
```

Comment:
rephrase to `cross shard transactions and rewards where destination shard is not the current node's shard won't be indexed in Elastic Search` as the es is the receiver here

Reply:
rephrased